.circleci/config.yaml

```diff
@@ -0,0 +1,31 @@
+version: 2.1
+orbs:
+  slack: circleci/slack@3.4.2
+
+jobs:
+  test:
+    working_directory: ~/circle
+    docker:
+      - image: cimg/ruby:2.7.2
+    steps:
+      - checkout
+      - setup_remote_docker
```

Comment:
is this needed?

Reply:
Good catch, no it is not needed :). I deleted already